docs/release-1.3/README.md

```diff
@@ -0,0 +1,309 @@
+    ```
+
+    During the installation process, Kubernetes cronjobs are created by the operator to monitor the lifecycle of the CRs managed by the operator: RHDH operator, OpenShift Serverless operator and OpenShift Serverless Logic operator. When deleting one of the previously mentioned CRs, a job is triggered that ensures the CR is removed before the operator is.
+    In case of any failure at this stage, these jobs remain active, facilitating administrators in retrieving detailed diagnostic information to identify and address the cause of the failure.
+
+    > **Note:** that every minute on the clock a job is triggered to reconcile the CRs with the orchestrator resource values. These cronjobs are deleted when their respective features (e.g. `rhdhOperator.enabled=false`) are removed or when the orchestrator resource is removed. This is required because the CRs are not managed by helm due to the CRD dependency pre availability to the deployment of the CR.
+
+1. Apply the Orchestrator custom resource (CR) on the cluster to create an instance of RHDH and resources of OpenShift Serverless Operator and OpenShift Serverless Operator Logic.
+   Make any changes to the [CR](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/config/samples/_v1alpha1_orchestrator.yaml) before applying it, or test the default Orchestrator CR:
+    ```console
+    oc apply -n orchestrator -f https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/refs/heads/main/config/samples/_v1alpha1_orchestrator.yaml
+    ```
+
+## Additional information
+
+### Additional Workflow Namespaces
+
+When deploying a workflow in a namespace different from where Sonataflow services are running (e.g., sonataflow-infra), several essential steps must be followed:
+
+1. **Label the Workflow Namespace:**
+  To allow Sonataflow services to accept traffic from workflows, apply the following label to the desired workflow namespace:
+   ```console
+      oc label ns $ADDITIONAL_NAMESPACE rhdh.redhat.com/workflow-namespace=""
+   ```
+2. **Identify the RHDH Namespace:**
+   Retrieve the namespace where RHDH is running by executing:
+   ```console
+      oc get backstage -A
+   ```
+   Store the namespace value in RHDH_NAMESPACE in the Network Policy manifest below.
+3. **Identify the Sonataflow Services Namespace:**
+   Check the namespace where Sonataflow services are deployed:
+   ```console
+      oc get sonataflowclusterplatform -A
+   ```
+   If there is no cluster platform, check for a namespace-specific platform:
+   ```console
+      oc get sonataflowplatform -A
+   ```
+   Store the namespace value in SONATAFLOW_PLATFORM_NAMESPACE.
+
+4. **Set Up a Network Policy:**
+   Configure a network policy to allow traffic only between RHDH, Sonataflow services, and the workflows. The policy can be derived from [here](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/templates/network-policies.yaml)
```

Comment:
Should be `https://github.com/parodos-dev/orchestrator-helm-operator/blob/release-1.3/helm-charts/orchestrator/templates/network-policies.yaml`